daemon/api.go

```diff
@@ -120,6 +121,12 @@ var (
 )
 
 func v1Get(c *Command, r *http.Request) Response {
+	lock, err := lockfile.Lock(dirs.SnapLockFile, true)
```

Comment:
Idle thought: As often with bools as arguments, "lockfile.Lock(file, true) " does not convey (to me at least) what the "true|false" here actually means. I wonder if it might make sense to add a "flag" to the lockfile api instead of bool and have lockfile.Lock(dirs.SnapLockFile, lockfile.BLOCKING)?